app/views/shared/_message_controls.html.haml

```diff
@@ -1,12 +1,12 @@
 = simple_form_for message, remote: true, authenticity_token: true  do |f|
```

Comment:
have created a tech debt pivotal ticket for this. https://www.pivotaltracker.com/story/show/113732351